dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/src/org/apache/cxf/ws/policy/PolicyEngineImpl.java

```diff
@@ -694,12 +706,18 @@ public boolean supportsAlternative(Collection<? extends PolicyComponent> alterna
 			       }
                                return true;
                             } else {
+                               if (doLog) {
+                                  LOG.fine("ignoreUnsupportedPolicy is not set, returning false 2");
```

Comment:
Liberty change flag?

Reply:
Not needed here, it's inside Start/End block